src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/SymbolEquivalenceComparer.GetHashCodeVisitor.cs

```diff
@@ -56,45 +56,26 @@ public int GetHashCode(ISymbol x, int currentHash)
             }
 
             private int GetHashCodeWorker(ISymbol x, int currentHash)
```

Comment:
📝 This file wasn't even part of the stack overflow review. It would have helped to submit it separately.